### ISSUE TITLE
__repr__ not supported in numba >0.56

### ISSUE THREAD

**Issue by knaaptime** [QuantEcon/Distributions.py#31]
with recent versions of numba, instantiating a class will raise [`Method '__repr__' is not supported.`](https://github.com/pysal/segregation/actions/runs/5732887274/job/16125324176#step:6:784). I think this is because `__repr__` is now set internally [by jitclass](https://github.com/numba/numba/issues/8533), and thus we can't [set it manually](https://github.com/QuantEcon/rvlib/blob/6a0b896fdebd86d67bf307f44f59fc3493dcea34/rvlib/univariate.py#L207). If I delete that repr definition, everything works (though the `str` representation gets wonky).

tl;dr since numba kinda wraps the `__repr__ ` and `__str__` now, i think it would probably be fine to just delete those definitions. The new ones coming from jitclass should work fine in most cases, and it seems unlikely folks would've built on the existing versions, so nothing will break if it changes.

The other solution would be to pin numba<0.56, but that seems less than ideal
